src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1050,23 +1169,6 @@ public bool IsTopMostNode
             }
         }
 
-        internal void ConnectInput(int inputData, int outputData, NodeModel node)
```

Comment:
The more I stare at this method, I don't think this effects the issue @ramramps and I are mentioning - those are other caches I believe. Sorry - I'll keep looking, its been a long time since I have checked this code out.